3-platform/order/infra/src/main/java/com/_6core/platform/orderinfra/adapter/driven/persistence/entity/OrderEntity.java

```diff
@@ -0,0 +1,31 @@
+package com._6core.platform.orderinfra.adapter.driven.persistence.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigInteger;
+import java.util.Set;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
```

Comment:
@Builder is redundant, we are not going to create a class on our own (only via mapstruct). 
Mapstruct can use @Setter and @Getter only. 

p.s. do you really desire to use Lombok? if yes, pls use such variant: 
```suggestion
@Getter
@Setter
@NoArgsConstuctor
```